products/sle/main.pm

```diff
@@ -695,6 +695,9 @@ sub load_consoletests() {
         loadtest "console/yast2_i";
         loadtest "console/yast2_bootloader";
         loadtest "console/vim";
+        if (check_var('UPGRADE', '1')) {
+            loadtest "console/sblim_sfcb";
```

Comment:
these consoletests you add it too runs on way more scenarios than you may imagine. And so e.g. the sleep 10 you think 'works for now' will add to each of them.

So we better add this to the extra tests and create a scenario to have them run in upgrades - which we don't' do atm.